trillian_log_api.proto

```diff
@@ -82,9 +82,9 @@ service TrillianLog {
   }
 
   // Returns the latest signed log root for a given tree. Corresponds to the
-  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  The server will
-  // return InvalidArgument if first_tree_size is greater than the
-  // LatestSignedLogRoot available to that server.
+  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  If
+  // first_tree_size > LatestSignedLogRoot, the server will return rpc code
```

Comment:
nit: "return rpc code Unavailable" is slightly vague &ndash; do you mean "return gRPC codes.Unavailable" ?

Reply:
Amended as requested.

Is gRPC implicit from the fact that this is a service definition in a proto file?